### ISSUE TITLE
chore(ionic-angular): migrate to @nrwl/devkit

### ISSUE THREAD

**Issue by devinshoemaker** [nxtend-team/nxtend#515]
- [ ] docs
  - [ ] remove init generator docs
  - [ ] update getting-started
    - [ ] remove init
    - [ ] update --help
- [ ] remove init generator
- [ ] remove @nrwl/workspace references
- [ ] replace join's with normalizePath
- [ ] format files

**Comment by JamesHenry** [comment]
Hi @devinshoemaker, thanks for all the work you do in the Nx community!

I have a client looking to prototype with Ionic in an existing Nx workspace leveraging distributed config (project.json), which I see you are also using yourself in this workspace.

When they run the application generator it blows up because the workspace.json that the Angular devkit host is reading is coming through as v1 (which naturally does not support project.json).

Naturally, this problem will be automatically taken care of by resolving this issue and completing the migration to nrwl/devkit, so I have not opened a new issue for it.

Is there existing WIP for this and/or would you like any help in getting it over the line?

**Comment by devinshoemaker** [comment]
Wow, thanks! That means a lot! I have been hard at work on migrating everything over to the devkit. I just about have all the plugins moved over, and then I just want to update the Ionic deps included in the plugins and it should be ready for a release. I have a couple days of work dedicated to this coming up, so I'm hoping to be close to the finish line by this weekend.

**Comment by JamesHenry** [comment]
Thanks so much for the incredibly prompt response, that's great to hear! We'll keep an eye out and be ready to pull in the latest changes once they are ready.

(This is another one that we observed (and I've had to remedy a couple of times myself in various plugins) and should hopefully be a pretty trivial one to include in those changes to the https://github.com/nxtend-team/nxtend/issues/516 in case it's not already included)

**Comment by devinshoemaker** [comment]
Yeah I'm still not sure why that issue occurs, but it seems to be resolved with the devkit migration when testing locally.
